src/mango_httpd.erl

```diff
@@ -50,8 +50,24 @@ handle_req_int(_, _) ->
 
 
 handle_index_req(#httpd{method='GET', path_parts=[_, _]}=Req, Db) ->
+    Params = lists:flatmap(fun({K, V}) -> parse_index_param(K, V) end,
+        chttpd:qs(Req)),
     Idxs = lists:sort(mango_idx:list(Db)),
-    JsonIdxs = lists:map(fun mango_idx:to_json/1, Idxs),
+    JsonIdxs0 = lists:map(fun mango_idx:to_json/1, Idxs),
+    % Set limit default to 200 here since dashboard defaults to 200 per page.
+    Limit = case couch_util:get_value(limit, Params, 200) of
```

Comment:
I think such implicit limit may cause a problems since no other API uses anything of such kind restrictions. Better leave no limits and let user define their own with request. Otherwise we'll end with a clients which will use magic values like `?limit=100500` just because they want to really fetch all the available indexes. Also, 200 is not a big number as it looks like.